java/code/src/com/redhat/rhn/frontend/xmlrpc/org/OrgHandler.java

```diff
@@ -314,7 +314,7 @@ private ChannelFamily lookupChannelFamily(String channelFamilyLabel) {
 
     /**
      * Migrate systems from one organization to another.  If executed by
-     * a Satellite administrator, the systems will be migrated from their current
+     * a product administrator, the systems will be migrated from their current
```

Comment:
```suggestion
     * a #product() administrator, the systems will be migrated from their current
```

But I am unsure if this macro works in normal Java

Reply:
> But I am unsure if this macro works in normal Java

I guess we could make it work, but currently - as far as I know - we are not actually building and publishing javadoc. And in general we would rather want to go away from using velocity, so I would probably not want to introduce this macro for being used in javadoc comments. Therefore I purposely chose to use just "product administrator" here, this way it should be valid in the context of Uyuni, but also downstream, while developers hopefully know what is meant (the current `SAT_ADMIN` aka `satellite_administrator` in the DB).

Additionally I had the idea to actually replace `satellite_admin` throughout the DB and codebase with `product_admin` (or maybe something else?) to be more generic. If we did this, that comment would not need to be updated. I would be curious to know your opinion about that?

Reply:
As nobody read this I am fine with keeping it.